diff --git a/CMakeLists.txt b/CMakeLists.txt
index ad8ded0..066c271 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -231,6 +231,11 @@ if (UNIX)
     FTCONFIG_H "${FTCONFIG_H}")
   file(WRITE "${PROJECT_BINARY_DIR}/include/freetype/config/ftconfig.h"
     "${FTCONFIG_H}")
+else ()
+  file(READ "${PROJECT_SOURCE_DIR}/include/freetype/config/ftconfig.h"
+    FTCONFIG_H)
+  file(WRITE "${PROJECT_BINARY_DIR}/include/freetype/config/ftconfig.h"
+    "${FTCONFIG_H}")
 endif ()
 
 
